README.md

```diff
@@ -24,6 +24,26 @@
 * Mapstruct
 * Lombok
 
+## Start the application using docker
+* download the app
+$ git clone https://github.com/KSM-development/wst-backoffice.git
```

Comment:
Please change all commands you started with $ to triple single quotation marks
E.g.
instead of 
$ git clone https://github.com/KSM-development/wst-backoffice.git

use this
\```
git clone https://github.com/KSM-development/wst-backoffice.git
\```

Reply:
done, it look greate